public/css/santaHistory.css

```diff
@@ -8,6 +8,7 @@
 
 .modal-body img {
 	width: 100%;
+	height: 600px;
```

Comment:
This should work with different screen sizes. 600px is pretty fixed and does not guarantee good overview.

Reply:
I don't know how it would look on mobile, but I'm pretty sure it will work on your screen.